episodes/files/plot_terse_amdahl_results.py

```diff
@@ -34,16 +44,28 @@ def process_files(file_list, output="plot.jpg"):
     # Show the legend
     plt.legend()
 
-    # Save the plot to a JPEG file
-    plt.savefig(output, format='jpeg')
+    # Save the plot to the specified file
+    plt.savefig(output, dpi=400, bbox_inches="tight")
 
 if __name__ == "__main__":
-    # The first command-line argument is the script name itself, so we skip it
-    output = sys.argv[1]
-    filenames = sys.argv[2:]
+    parser = argparse.ArgumentParser(
+        description=description,
+        epilog="Brought to you by HPC Carpentry"
+    )
+
+    parser.add_argument(
+        "--output",
+        default="scaling-study.png",
+        help="Image file to write (PNG or JPG)",
+        required=True
```

Comment:
You're right! I think `--output` should be a required named argument; I'll drop the default. Otherwise, there's too much ambiguity, and -- perhaps more importantly -- with only positional arguments, the script returns an error if run without any. Requiring a named input triggers the help/usage message to run if you just invoke `./plot_terse_amdahl_results.py`.